File: app/src/mockApi/java/org/example/product/app/mock/api/MockServer.java
package org.example.product.app.mock.api;

import com.google.common.collect.ImmutableList;
import org.example.product.app.MainModule;

public class MockServer {

    public MockServer() {}

    public ImmutableList<String> get() {
        new MainModule();
        return ImmutableList.of("a", "b", "x", "y");
    }
}

File: app/src/mockApi/kotlin/org/example/product/app/mock/api/MockServer.kt
package org.example.product.app.mock.api

import com.google.common.collect.ImmutableList
import org.example.product.app.MainModule

class MockServer {
    fun get(): ImmutableList<String> {
        MainModule()
        return MockServer.get()
    }

    companion object {
        fun get(): ImmutableList<String> {
            return ImmutableList.of("a", "b", "x", "y")
        }
    }
}
